guidedremediation/upgrade/upgrade.go

```diff
@@ -26,6 +26,7 @@ import (
 // if Level == Minor, only upgrades up to minor (1.0.0 - 1.*.*) are allowed.
 type Level int
 
+// these constants represent something
```

Comment:
```suggestion
// these constants represent the different allowable semver upgrade levels
```

This comment feels redundant with the one on `type Level int` above.
Is there a consensus on what to comment on the enum type versus the values?